docs/openapi3.yaml

```diff
@@ -23,7 +23,7 @@ paths:
             application/json:
               schema:
                 allOf:
-                  - $ref: 'http://localhost/Schema/Http/ImageMetadataResponse.json'
+                  - $ref: '../Schema/Http/ImageMetadataResponse.json'
```

Comment:
1. do you think we need to rename to camel case?
2. why not ImageMetadata-Response as with the ImageMetadata-Create and -Update?

Reply:
1) don't think it matters
2)that's the name of the file in models
2.1) the -create and -update in the models takes a base model and only set what fields are required. this is a composition of 2 different types